lib/coverband/utils/source_file.rb

```diff
@@ -139,7 +146,7 @@ def build_lines
         coverage_exceeding_source_warn if coverage.size > src.size
 
         lines = src.map.with_index(1) { |src, i|
-          Coverband::Utils::SourceFile::Line.new(src, i, never_loaded ? 0 : coverage[i - 1])
+          Coverband::Utils::SourceFile::Line.new(src, i, never_loaded ? 0 : coverage[i - 1], never_loaded ? nil : coverage_posted[i - 1])
```

Comment:
We may should check whether coverage_posted is nil because when not using hash redis store, it will be nil.

When not using hash redis store, the coverage server crashes.
```
Puma caught this error: undefined method `[]' for nil:NilClass (NoMethodError)
/home/vscode/src/github.com/C-FO/coverband/lib/coverband/utils/source_file.rb:150:in `block in build_lines'
...
```
```suggestion
          Coverband::Utils::SourceFile::Line.new(src, i, never_loaded ? 0 : coverage[i - 1], never_loaded || !coverage_posted.is_a?(Array) ? nil : coverage_posted[i - 1])
```